nixos/modules/i18n/input-method/ibus.nix

```diff
@@ -55,7 +55,7 @@ in
 
     # Without dconf enabled it is impossible to use IBus
```

Comment:
Unrelated to this change, but I'm certain that what should have been done here is
```
services.dbus.packages = [ pkgs.gnome3.dconf ];
```

since `config.system.path` gets added to that option automatically.